### PR TITLE
Nextcloud install polish + remove correct nginx config

### PR DESCRIPTION
**Pull request by MichaIng**
+ adjust all php variables together at the top

### CODE REVIEW

**Comment by MichaIng** [comment]
And do nginx php settings work now?

File: dietpi/dietpi-software [hunk 1]
```diff
@@ -258,12 +258,12 @@ _EOF_
...
-	FP_PHP_BASE_DIR='/etc/php5'
-	#	PHP- or PHP5- apt version names for installation.
+	#PHP5/7 specific directories, apt package-, module- and command names
+	FP_PHP_BASE_DIR='/etc/php/7.0'
 	PHP_APT_PACKAGE_NAME='php'
 	if (( $DISTRO == 3 )); then
-
+		
+		FP_PHP_BASE_DIR='/etc/php5'
 		PHP_APT_PACKAGE_NAME='php5'
```
Comment (Fourdee) [inline dietpi/dietpi-software:267]: Nice :)

**Comment by Fourdee** [comment]
@MichaIng 

Nice one 👍.

I'll get this merged and re-test

**Comment by Fourdee** [comment]
For reference: https://github.com/Fourdee/DietPi/issues/1067#issuecomment-335200352

**Pull request merged**